### DISCUSSION TITLE
1.2.8 (and 1.2.9) has been released!

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#8655] [Announcements]
Happy Thursday everyone! [We just released version 1.2.8](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.8), which primarily just has a lot of smaller back end optimizations and improvements for our own internal needs.

That said, it's *very* important folks update to this version as soon as feasible, if you have not already updated to version 1.2.6 or higher. As we announced several weeks ago with the release of 1.2.6, if you are running a version older than that, it is possible for your node to experience issues if a high number of blocks are compressed by Blue Box Timelords. We are ramping up a large number of Blue Boxes over the coming days/weeks temporarily to help compact the chain, so now is a great time to update if you haven't already!

Edit: And [we just updated to 1.2.9 ](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.9)as well with a minor fix to the incorrect version numbers that are appearing on certain platforms. If you aren't experiencing a version number issue, or simply don't care if it shows a dev version number, you can safely ignore this update and catch it's changes in the next release. :)

**Comment by ArIulzZz** [comment]
Shows 1.2.6dev67 and there's no new translations to GUI which means it doesn't installing the 1.2.8 version !
Can someone fix ? Ubuntu Clone + Windows Installer - Both Provides 1.2.6dev67 !!!!!!!!!!!

**Reply by sargonas** [reply]
You *technically* do have the latest version if it is 1.2.6dev67, as there was a bug causing the version numbers to not display as expected and return a dev number instead. We just released 1.2.9 which will fix this for you, however!

**Comment by minfengdiannao** [comment]
自动1.2.3版本后，经常会弹出这种日志：WARNING  Have multiple copies of the plot，实际上并没有重复的绘图，必须要重新启动软件，以重新读图，这个BUG已经出现了很久，但一直没有得到修复！
